modules/nf-core/stitch/main.nf

```diff
@@ -0,0 +1,83 @@
+process STITCH {
+    tag "$meta.id"
+    label 'process_high'
```

Comment:
Is there a particular reason why you use process_high?

Reply:
In general it is very resource-heavy as a method, probably would be the heaviest step in most pipelines. Do you think would be better to use something different? The memory/time scale quadratically with number of haplotypes (K), and I think approximately linearly with number of SNPs and number of samples

Reply:
Also memory scales ~linearly with number of threads

Reply:
Regarding the process tag configuration.
Do you think it's feasible to run it with less than 12 CPU, 84 Gb in 10 hours ?

Reply:
Depends on the sample size and parameters, for what I personally do with it no, but with small sample sets I think yes

Reply:
@nf-core/core Could we have a bit of help here ?
We would like to know what's the best to do when the process tag "medium" is maybe a bit limit to work.

Reply:
I think we can set it to medium then and the user can override as required, there can be use cases where medium is enough I guess